lib/sageone_sdk/authentication.rb

```diff
@@ -4,5 +4,39 @@ module Authentication
     def token_authenticated?
       !!@access_token
     end
+
+    def authorization_url
+      "https://www.sageone.com/oauth2/auth?response_type=code&client_id=#{@client_id}&redirect_uri=#{redirect_uri}&scope=full_access"
```

Comment:
Is this url correct? I'd have expected `app.sageone.com` opposed to `www.sageone.com`